galter_subjects_utils/types_internal.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
types_internal.py could be renamed to subject.py, but I had the Analysis class in and out of this file in the process of working on this. Plus replacements may get their own class in the future... Leaving it open like this for now and we'll see...